packages/utils/src/getDateElementProps.ts

```diff
@@ -0,0 +1,45 @@
+import { type DateObject } from './types';
+
+export type DateElementFormat = 'DMY' | 'MDY' | 'YMD';
+
+type DateElementProp = {
+  type: string;
+  range: [number, number];
+  value: number | undefined;
+};
+
+export default function getDateElementProps(
```

Comment:
Can you please add documentation to this function as well as replicating that documentation into the official RJSF online documentation found here `packages/docs/docs/api-reference/utility-functions.md`

Reply:
Hi, @heath-freenome 

I have updated my PR with the required changes.